drivers/google/google.go

```diff
@@ -19,6 +19,7 @@ type Driver struct {
 	SSHPort        int
 	Zone           string
 	MachineType    string
+	DiskTypeURL    string
```

Comment:
Sorry to be nitpicking, but why is this called `diskTypeURL`?  Shouldn't we be just specifying the type?